### ISSUE TITLE
Rename OpsSight Internal Registry File Path

### ISSUE THREAD

**Issue by mphammer** [blackducksoftware/synopsys-operator#293]
**What would you like to be added**:
JIRA Ticket CLOUD-369

```
let's rename this opssight param:
--scannerpod-imagefacade-internal-registries-file-path string Absolute path to a file for secure docker registries credentials to pull the images for scan
to:
--scannerpod-imagefacade-secure-registries-file-path string Absolute path to a file for secure docker registries credentials to pull the images for scan
```

**Comment by yashbhutwala** [comment]
Fixed by #295

**Comment by rrati** [comment]
verified by passing args to opssight create command and verified it attempted to read the file location.
